### PR TITLE
Pub 1160 accessibility statement page

### PR DESCRIPTION
**Pull request by joshblackmoor**
### JIRA link ###

https://tools.hmcts.net/jira/browse/PUB-1160

### Change description ###

Created the accessibility statement page and updated the footer to direct to the new page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/views/accessibility-statement.njk [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+{% endblock %}
+
+{% block beforeContent %}
+  {{ super() }}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">{{ title }}</h1>
+
+  {# Section 1 #}
```
Comment (kurt-uk) [inline src/main/views/accessibility-statement.njk:14]: Could wrap sections in id'd divs to help section areas

File: src/test/unit/views/accessibility-statement.test.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  });
+
+  it('should display the heading in section 2', () => {
+    const heading = htmlRes.getElementsByClassName(mediumHeadingClass);
+
+    expect(heading[0].innerHTML).contains('How accessible is this website?', 'Could not find section 2 heading');
+  });
+
+  it('should display the heading in section 3', () => {
+    const heading = htmlRes.getElementsByClassName(mediumHeadingClass);
```
Comment (kurt-uk) [inline src/test/unit/views/accessibility-statement.test.ts:34]: the id's would make selecting them more readable too

**Review by kurt-uk — CHANGES_REQUESTED** [review]

File: src/main/views/accessibility-statement.njk (hunk 1 above)
Comment (joshblackmoor) [reply src/main/views/accessibility-statement.njk]: Good point, done :)

File: src/main/routes/routes.ts [hunk 3]
```diff
@@ -106,6 +106,7 @@ export default function(app: Application): void {
   app.get('/summary-of-publications', app.locals.container.cradle.summaryOfPublicationsController.get);
   app.get('/file-publication', app.locals.container.cradle.flatFileController.get);
   app.get('/sjp-public-list', app.locals.container.cradle.sjpPublicListController.get);
+  app.get('/accessibility-statement', app.locals.container.cradle.accessibilityStatementController.get);
```
Comment (junaidiqbalmoj) [inline src/main/routes/routes.ts:109]: Very small change :) Can you alphabetically arrange new route? It should come after app.get('/', app.locals.container.cradle.homeController.get);

Comment (joshblackmoor) [reply src/main/routes/routes.ts]: Done :)

**Review by junaidiqbalmoj — APPROVED** [review]

**Review by kurt-uk — APPROVED** [review]
nice work

File: src/main/resources/locales/en/accessibility-statement.json [hunk 4]
```diff
@@ -0,0 +1,108 @@
+{
+  "title": "Accessibility statement for the Court and Tribunal Hearing Information Service",
+  "section1": {
+    "text1": "The Court and Tribunal Hearing Information Service provides a publishing platform to enable information provided by HMCTS to be shared, displayed and updated as appropriate. It will enable lists, outcomes, judgments and information to be displayed to court users, consistent with jurisdictional procedures and business rules. The service supports HMCTS’ commitment to open justice and the provision of transparent and consistent court/tribunal information across all jurisdictions.",
+    "text2": "This accessibility statement applies to content published on ",
+    "text3": " It is based on an accessibility audit for carried out by the Digital Accessibility Centre (DAC) user/ technical team on 1 March 2022.",
```
Comment (ChrisS1512) [inline src/main/resources/locales/en/accessibility-statement.json:6]: I think the spacing on user/ technical here needs updating too (same fix as the one further down)

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/main/resources/locales/en/accessibility-statement.json (hunk 4 above)
Comment (joshblackmoor) [reply src/main/resources/locales/en/accessibility-statement.json]: Done :)

**Review by ChrisS1512 — DISMISSED** [review]

**Review by furnivall — DISMISSED** [review]
:)

**Review by ChrisS1512 — APPROVED** [review]

**Review by junaidiqbalmoj — APPROVED** [review]

**Pull request merged**